components/Home/Home.vue

```diff
@@ -71,6 +73,66 @@ const device = useDevice()
 //
 // Hooks
 //
+onBeforeMount(async () => {
+  const favs = getHashPart(router, 'favs')
+  if (favs) {
+    try {
+      const addressHashes: string[] = []
+
+      favs
+        .split(',')
+        .forEach((id) => {
+          if (!Number.isNaN(Number(id)))
+            favoritesStore.toggleFavorite(Number(id))
+
+          if (id.startsWith('addr-'))
```

Comment:
Can you use `addr:` as prefix. To homogeneous with `ref:` and other `:` was have in the API.